test/DocumentFormat.OpenXml.Packaging.Tests/OpenXmlPackageTests.cs

```diff
@@ -301,5 +304,31 @@ public void TestOpenModel3DWrittenByPowerPoint_DashMime()
 
             Assert.Equal("model/gltf-binary", model3DReferenceRelationshipPart.ContentType);
         }
+
+        // Test issue #1281 for regressions.
```

Comment:
Using issue numbers in code is not helpful - the name of the test should be sufficient. If you feel it's really needed, use a full URI to the issue so it can be found easier

Reply:
removed